src/Domain/Creators/BinaryCompactObject.hpp

```diff
@@ -316,6 +316,14 @@ class BinaryCompactObject : public DomainCreator<3> {
     static constexpr Options::String help = {"Radius of the entire domain."};
   };
 
+  struct OpeningAngle {
+    using group = OuterShell;
+    static std::string name() { return "OpeningAngle"; }
+    using type = double;
+    static constexpr Options::String help = {
+        "The opening angle of the two half wedges in degrees."};
```

Comment:
Maybe be a bit more descriptive? I don't think a user necessarily knows what the half wedges are. Maybe something like

```
Surrounding the two objects (which are encompassed by two cubes) are a series of wedges that abut the
 faces of these cubes. The wedges on the ends of these cubes are full wedges while the wedges around 
the rest of the faces are half wedges. This option is the opening angle of these half wedges in degrees.
```

If I said something incorrect here you can just fix it when you add it.

Reply:
I don't want to make the help string too long and since this option is placed in `group = OuterShell` and the outer shell is in the main BCO documentation I can change `half wedges` to `half wedges of the outer shell` and the user should have enough information to find out what this is referring to with this string. But I will also add documentation on the half wedges in that place!

Reply:
Ok yeah that sounds good.